samtranslator/model/eventsources/pull.py

```diff
@@ -64,6 +64,8 @@ def to_cloudformation(self, **kwargs):
         lambda_eventsourcemapping.StartingPosition = self.StartingPosition
         lambda_eventsourcemapping.BatchSize = self.BatchSize
         lambda_eventsourcemapping.Enabled = self.Enabled
+        if 'Condition' in function.resource_attributes:
```

Comment:
If would be awesome to have a way to do this as a function and not need to worry about it in every resource. As we add more Events, resources, etc, this can be very easily forgotten.